README.md

```diff
@@ -43,36 +44,61 @@ Ensure you have the following installed:
 3. **[Docker](https://docs.docker.com/engine/install/)**
 4. **Operation System**: Ubuntu/MacOS
 
-### Installation
+### Setup
 
-1. Clone the repository and install the required dependencies.
+1. Clone the repository
 
-   ```bash
-   git clone https://github.com/ONSdigital/dis-wagtail.git
-   ```
+    ```bash
+    git clone https://github.com/ONSdigital/dis-wagtail.git
+    ```
 
 2. Install dependencies
 
-   [Poetry](https://python-poetry.org/) is used to manage dependencies in this project. For more information, read
-   the [Poetry documentation](https://python-poetry.org/).
+    [Poetry](https://python-poetry.org/) is used to manage dependencies in this project. For more information, read
+    the [Poetry documentation](https://python-poetry.org/).
+
+    To install all dependencies, including development dependencies, run:
+
+    ```bash
+    make install-dev
+    ```
+
+    To install only production dependencies, run:
+
+    ```bash
+    make install
+    ```
+
+3. Build the docker container
+
+    ```bash
+    make docker-build
+    ```
 
-   To install all dependencies, including development dependencies, run:
+4. Run the docker container
 
-   ```bash
-   make install-dev
-   ```
+    ```bash
+    make docker-start
+    # ssh into the web container
+    make docker-shell
+    # in the web container
+    $ djrun
```

Comment:
Is this missing a step somewhere to apply migrations?

Reply:
ah yes. thank you for checking. needs `./manage.py migrate` (or `dj migrate` as the alias)